src/main.ts

```diff
@@ -15,41 +15,43 @@ const defaults: ConfigEntry[] = [
 export const context = github.context;
 
 export async function run() {
-  try {
-    const repoToken: string = core.getInput("repo-token", { required: true });
-
-    if (context && context.payload && context.payload.repository && context.payload.pull_request) {
-      const octokit = new github.GitHub(repoToken);
-      const repoConfig: ConfigEntry[] = await getConfig(octokit, CONFIG_FILENAME, context.repo);
-      const config: ConfigEntry[] = repoConfig.length > 0 ? repoConfig : defaults;
-      const headRef = context.payload.pull_request.head.ref;
-      const baseRef = context.payload.pull_request.base.ref;
-      const labelsToAdd = config.reduce((labels: string[], entry) => {
-        if (entry.head && entry.base) {
-          if (isMatch(headRef, entry.head) && isMatch(baseRef, entry.base)) {
-            labels.push(entry.label);
-          }
-        } else if (entry.head && isMatch(headRef, entry.head)) {
-          labels.push(entry.label);
-        } else if (entry.base && isMatch(baseRef, entry.base)) {
+  const repoToken: string = core.getInput("repo-token", { required: true });
+
+  core.debug(`context: ${context ? JSON.stringify(context) : ''}`);
+
+  if (context && context.payload && context.payload.repository && context.payload.pull_request) {
+    const octokit = new github.GitHub(repoToken);
+    const repoConfig: ConfigEntry[] = await getConfig(octokit, CONFIG_FILENAME, context);
+    core.debug(`repoConfig: ${JSON.stringify(repoConfig)}`);
+    const config: ConfigEntry[] = repoConfig.length > 0 ? repoConfig : defaults;
+    core.debug(`config: ${JSON.stringify(config)}`);
+    const headRef = context.payload.pull_request.head.ref;
+    const baseRef = context.payload.pull_request.base.ref;
+    const labelsToAdd = config.reduce((labels: string[], entry) => {
+      if (entry.head && entry.base) {
+        if (isMatch(headRef, entry.head) && isMatch(baseRef, entry.base)) {
+          core.debug(`Matched "${headRef}" to "${entry.head}" and "${baseRef}" to "${entry.base}". Setting label to "${entry.label}"`);
```

Comment:
I think this log message would also be useful in "production", so setting the log level to `info` won't hide it according to https://github.com/actions/toolkit/tree/main/packages/core#logging
```suggestion
          core.info(`Matched "${headRef}" to "${entry.head}" and "${baseRef}" to "${entry.base}". Setting label to "${entry.label}"`);
```